src/main/scala/mesosphere/marathon/DeprecatedFeatures.scala

```diff
@@ -47,13 +47,7 @@ object DeprecatedFeatures {
     softRemoveVersion = SemVer(1, 7, 0),
     hardRemoveVersion = SemVer(1, 8, 0))
 
-  val kamonMetrics = DeprecatedFeature(
```

Comment:
Shouldn't it be kept, so that users get more descriptive error messages?

Reply:
Good catch. Reverted